partner_data_vies_populator/models/res_partner.py

```diff
@@ -0,0 +1,53 @@
+# Copyright (C) 2015 Forest and Biomass Romania
+# Copyright (C) 2020 NextERP Romania
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+import logging
+
+from odoo import _, api, models
+
+_logger = logging.getLogger(__name__)
+
+try:
+    from stdnum.eu.vat import check_vies
+except ImportError:
+    _logger.debug("Cannot import check_vies method from python stdnum.")
+
+
+class ResPartner(models.Model):
+    _inherit = "res.partner"
+
+    @api.model
+    def _get_vies_data(self, vat):
+        res = {}
+        vat = vat.strip().upper()
+        vat_country, vat_number = self._split_vat(vat)
+        result = check_vies(vat)
```

Comment:
This is failing if the country code is not european. One possible solution to do this before `check_vies`.
```python
try:
    compacted_vat = compact(vat)
except InvalidComponent:
    # This not an european company, so we are not processing it
    return {}
```

Reply:
`compact` and `InvalidComponent` are imported as `from stdnum.eu.vat import check_vies, compact, InvalidComponent`

Reply:
Update it checking the country code in base europe group